### ISSUE TITLE
Chisel modes don't work in Creative Mode

### ISSUE THREAD

**Issue by DrGamble** [ChiselsAndBits/Chisels-and-Bits#1094]
I have encountered an issue with the ChiselsAndBits Mod.
Some modes won't work if you play on creative mode when using a new chisel.
I have installed the mod using the forge installer and the standard client drag/drop method.
I have run this in an isolated instance of Minecraft 1.19.2 to see if it is potentially due to cross-mod incompatibility, but the bug remains.

#### To Reproduce:

1. Play on Creative
2. Acquire a new chisel of any type
3. Try to use on any block with any of the affected modes listed below

<br>

#### workaround:

1. Play on any game mode
2. Acquire a new chisel of any type
4. Switch to Survival (if not already on it)
5. Use once on any chisel mode to apply the item durability bar to chisel
6. Switch to Creative
7. Use the chisel

<br>

#### Affected Modes:

- Extrude 
    - all modes work 
    - gives a message "The state you are trying to place is not valid."
- Connected Plane 
    - all modes work
    - gives a message "The state you are trying to place is not valid."
- Plane
    - all modes work
    - gives a message "The state you are trying to place is not valid." 

- Drawn Cube
    - no modes work
    - no message was given
- Line
    - no modes work
    - no message was given

---

#### Mod and Minecraft information:

- Minecraft version: 1.19.2
- Forge? Yes
    - Forge Version 43.2.8
    - Scena 1.0.93
- Optifine? No
- ChiselsAndBits version: 1.3.129

---

 ### Additional references:

Related issues:
   - #1086
        - Closed by the poster
        - The issue was not resolved
   - #931
        - Closed by the dev
        - Stated it was fixed in ChiselsAndBits v1.2.100

**Comment by marchermans** [comment]
Did you install the mod with the curse forge launcher?

**Comment by DrGamble** [comment]
No, I installed it by dropping the jar into the mods folder that's created after installing forge

**Comment by marchermans** [comment]
Then I will need the debug.log file to check what is going on.

**Comment by DrGamble** [comment]
[debug.log](https://github.com/ChiselsAndBits/Chisels-and-Bits/files/11132483/debug.log)

**Comment by marchermans** [comment]
Okey. From your log it is indicated that there are some other mods loaded. Can you check if this happens with just c and b?

**Comment by DrGamble** [comment]
[debug.log](https://github.com/ChiselsAndBits/Chisels-and-Bits/files/11141485/debug.log)
Okay, I created a new installation with just the C&B mod, but the issue remains.

**Comment by marchermans** [comment]
How do you install forge?

**Comment by DrGamble** [comment]
I used the forge site 
https://files.minecraftforge.net/net/minecraftforge/forge/

**Comment by DrGamble** [comment]
Specifically thru the site's downloadable installer.

**Comment by marchermans** [comment]
Sorry but I can not reproduce this.
If I install those exact versions it works for me without a hitch.
There must be something in your system that prevents it from working.

**Comment by AlisonHuang777** [comment]
I also experience this issue despite under different versions, with Minecraft version 1.20.1, Forge version 47.1.44, C&B version 1.4.141, and JEI version 15.2.0.25. No other mods are installed.

**Comment by AlisonHuang777** [comment]
By the way, what does "extrude" chisel mode actually do? It seems not working properly for me. (It acts exactly like "plane" mode) Any footage please? Thanks in advance.

**Comment by Dickovski** [comment]
Having the same sort of issues.  Tried the work work around suggested of going to survival, using a chisel, and then returning to creative.  Now the line mode works, which it wasn't before,  but everything is a bit off from where it should be, the bits come out a few bits down.  Haven't tried starting Minecraft with only Chisels & Bits loaded, I'll have a go at this tomorrow.
